CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add `lifestage-context` note to `nursing-internal`
 * Add section titles links to `summary` in `nursing-internal`
 * Change EOC titles in `nursing-internal`
+* Added `index` and `appendix` to `neuroscience`
```

Comment:
Sorry, first I approved but then I noticed problem with CHANGELOG. Please move this log to `Unreleased`. Versions that are already released shouldn't be changed.